File: Code/05_File_POSIX/Ex3.c
#include <stdio.h>
#include <stdlib.h>
#include <fcntl.h>
#include <unistd.h>

int main(int argc, const char * argv[])
{
	int fd = open("exercicio3.txt",
		O_RDWR | O_CREAT);
	if(fd==-1)
	{
		printf("Erro na abertura"
			" do arquivo.\n");
		exit(-1);
	}
	printf("Arquivo 'exercicio3.txt'"
		" criado. Detalhes:\n");
	system("ls -l exercicio3.txt");
	close(fd);
	return 0;
}

File: 2_Programação_sistemas_operacionais/2.2_File_POSIX/Python/Ex3.py
import os

try:
	fd = os.open("exercicio3.txt", os.O_RDWR | os.O_CREAT)
except OSError:
	print("Erro na abertura do arquivo.")
	exit(-1)
print("Arquivo 'exercicio3.txt' criado. Detalhes:")
os.system("ls -l exercicio3.txt")
os.close(fd)